docs/for_continue_increment.md

```diff
@@ -0,0 +1,17 @@
+# For Continue Increment
+
+## Configuration
+* Check: `for-continue-increment`
```

Comment:
Please change the check on `pess-for-continue-increment` to make it distinguishable from the original detectors